src/js/tabs/firmware_flasher.js

```diff
@@ -29,6 +29,7 @@ firmware_flasher.initialize = function (callback) {
     self.parsed_hex = undefined;
 
     const unifiedSource = 'https://api.github.com/repos/betaflight/unified-targets/contents/configs/default';
+    const divReleaseInfo = 'div.release_info';
```

Comment:
Does the element `$('div.release_info)` exist at this point?

If yes, usually we create directly the element:
```javascript
const divRealeaseInfoElement = $('div.release_info');
```
and we use it later in the rest of the code.

Reply:
The element is not available at that point, that's why I choose this solution to the solve sonar error.

Reply:
Found out something weird. Using the $(selector) as a constant was not evaluated at the places we are using them.
I mean when using $('selector') it works not when it's assigned to a constant :confused: